### PR TITLE
feat!: Update prql-python to 0.11

### PR DESCRIPTION
**Pull request by max-sixty**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -19,17 +19,17 @@ license = "Apache-2.0"
 name = "pyprql"
 readme = "README.md"
 repository = "https://github.com/prql/pyprql"
-version = "0.10.0"
+version = "0.11.0"
```
Comment (eitsupi) [inline pyproject.toml:22]: Do not manually increase the version. The version should be determined automatically by CI.

**Review by eitsupi — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (max-sixty) [reply pyproject.toml]: Thanks, corrected

**Review by eitsupi — APPROVED** [review]
Thanks.

**Comment by max-sixty** [comment]
FYI I started the GHA: https://github.com/PRQL/pyprql/actions/runs/7283781328

**Comment by eitsupi** [comment]
Thanks, I created the release.

**Comment by max-sixty** [comment]
> Thanks, I created the release.

Thanks!

**Pull request merged**